code/controllers/subsystem/ticker.dm

```diff
@@ -711,7 +711,8 @@ SUBSYSTEM_DEF(ticker)
 		'sound/roundend/yeehaw.ogg',
 		'sound/roundend/disappointed.ogg',
 		'sound/roundend/gondolabridge.ogg',
-		'sound/roundend/haveabeautifultime.ogg'\
+		'sound/roundend/haveabeautifultime.ogg',
+		'sound/roundend/CitadelStationHasSeenBetterDays'\
```

Comment:
did you test this PR?

Reply:
Thanks for reminding me since I felt something was off and this helped point it out to me.